docs/src/rules/no-trailing-spaces.md

```diff
@@ -58,7 +58,7 @@ Examples of **correct** code for this rule with the `{ "skipBlankLines": true }`
 
 var foo = 0;
 var baz = 5;
-//•••••
+// trailing whitespace →     
```

Comment:
This example is "correct" in the docs but "incorrect" in the playground. I think this should be a line with spaces only so that `"skipBlankLines": true` has an effect.

Reply:
Fixed, thanks! I forgot to check this block.